src/amo/components/AddonReviewListItem/styles.scss

```diff
@@ -1,15 +1,30 @@
 @import '~photon-colors/photon-colors';
+@import '~core/css/inc/vars';
 @import '~core/css/inc/mixins';
 @import '~amo/css/inc/vars';
 
+.AddonReviewListItem {
+  .Rating {
+    align-self: start;
```

Comment:
This doesn't seem to be needed. Changing it doesn't have any impact on the position of the rating stars. Is it there "just in case"?

Reply:
Huh. I guess it's not needed anymore, I'll remove it. At one point it solved the vertical centering shown in the *before* screenshot up above but that's not a problem anymore.